core/plugins/org.polarsys.capella.core.data.core.validation/plugin.xml

```diff
@@ -593,6 +593,14 @@ This rule checks that an capella element doesn&apos;t contain any naming conflic
                      class="Involvement">
                </target>
             </constraint>
+         </constraints>
+      </constraintProvider>
+      <constraintProvider>
+         <package
+               namespaceUri="http://www.polarsys.org/capella/common/core/5.0.0">
```

Comment:
can you move the rule at L525 to avoid two same namespaceUri in the plugin.
https://github.com/eclipse/capella/pull/448/files#diff-3d2ada7e8983db4d55302278c19145d5d1bab055230189911dc936ce29e52d6eR523

Reply:
Done